mantidproject/mantid
# 37376  texts on the mantid workbench main menu could not be seen on osx dark mode

### Description of work

#### Summary of work
 The text on the mantid workbench main menu could not be seen on osx dark mode. The fix includes the QComboBoxes in the starting page.

 Make text visible on OSX Dark mode.

Fixes #37376. 


  **Report to:** @yusufjimoh, @jhaigh0

#### Further detail of work


### To test:



1.  Run Mantid workbench using MacOS on Dark Mode, and make all text is visible. 
2. Check that the drop down boxes appear to have a white background and black text when viewed.
Preferably: Test on MacOs with Dark Mode off as well.

 Adding release notes after this PR



---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
